src/main/java/org/isf/therapy/gui/TherapyEdit.java

```diff
@@ -604,7 +599,7 @@ private JCheckBox getSmsCheckBox() {
 			smsCheckBox.setEnabled(false);
 			smsCheckBox.setAlignmentX(CENTER_ALIGNMENT);
 			smsCheckBox.setAlignmentY(CENTER_ALIGNMENT);
-			if (!GeneralData.SMSENABLED || admitted) {
+			if (!GeneralData.SMSENABLED) {
```

Comment:
Can this be simplified to `smsenable = GeneralData.SMSENABLED` or the `smsenable` variable can be removed completely and use the `GeneralData.SMSENABLED` variable.

Reply:
Yep!